usage/app/lib/EventProcessor.scala

```diff
@@ -0,0 +1,100 @@
+package lib
+
+import java.util.{ List => JList }
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.contentapi.client.model.ItemQuery
+import play.api.Logger
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import com.amazonaws.services.kinesis.clientlibrary.interfaces.{IRecordProcessorCheckpointer, IRecordProcessor}
+import org.joda.time.DateTime
+import com.amazonaws.services.kinesis.clientlibrary.types.ShutdownReason
+import com.amazonaws.services.kinesis.model.Record
+import com.gu.thrift.serializer.ThriftDeserializer
+import com.gu.crier.model.event.v1.{EventPayload, Event, EventType}
+
+
+
+private class CrierEventProcessor() extends IRecordProcessor {
+
+    override def initialize(shardId: String): Unit = {
+      Logger.debug(s"Initialized an event processor for shard $shardId")
+    }
+
+    override def processRecords(records: JList[Record], checkpointer: IRecordProcessorCheckpointer): Unit = {
+
+      records.asScala.map { record =>
+
+        val buffer: Array[Byte] = record.getData.array()
+
+        for {
+          result: Event <- CrierDeserializer.deserialize(buffer, false)
+        } yield {
+
+          processEvent(result)
+        }
+
+      }
+    }
+
+    override def shutdown(checkpointer: IRecordProcessorCheckpointer, reason: ShutdownReason): Unit = {
+      if (reason == ShutdownReason.TERMINATE) {
+        checkpointer.checkpoint()
+      }
+    }
+
+    private def processEvent(event: Event): Unit = {
+
+      val dateTime: DateTime = new DateTime(event.dateTime)
+
+      event.eventType match {
+        case EventType.Update => {
+
+          event.payload match {
+            case Some(content: EventPayload.Content) => {
```

Comment:
We don't at the moment as no one has expressed a need for them, but are happy to consider others if requested. Something like Fronts doesn't exists within CAPI so would not be something we personally would provide.